docs/dev/ios/debug-glean-on-ios.md

```diff
@@ -0,0 +1,49 @@
+# Debug an iOS application against different builds of Glean
+
+At times it may be necessary to debug against a local build of Glean or another git fork or branch in order to test new features or specific versions of Glean.
+
+Since Glean is consumed through [Carthage](https://github.com/Carthage/Carthage), this can be as simple as modifying the Cartfile for the consuming application.
+
+## Building against the latest Glean
+
+For consuming the latest version of Glean, the Cartfile contents would include the following line:
+
+```
+github "mozilla/glean" "master"
+```
+
+This will fetch and compile Glean from the [mozilla/glean GitHub](https://github.com/mozilla/glean/) repo from the "master" branch.
```

Comment:
```suggestion
This will fetch and compile Glean from the [mozilla/glean GitHub](https://github.com/mozilla/glean/) repository from the "master" branch.
```